src/Makefile.in

```diff
@@ -855,6 +855,9 @@ clean:
 	@for exe in ${PROJ}; do \
 	  ($(RM) $$exe.exe) \
 	done
+	@for object in ${JOHN_OBJS}; do \
+	  ($(RM) $$object) \
+	done
```

Comment:
This is probably wrong. I'm not seeing what I expect to see during `make clean`.